test/blockwisealgorithms/CMakeLists.txt

```diff
@@ -1,4 +1,14 @@
 VIGRA_CONFIGURE_THREADING()
-VIGRA_ADD_TEST(test_blockwiselabeling test_labeling.cxx)
-VIGRA_ADD_TEST(test_blockwisewatersheds test_watersheds.cxx)
-VIGRA_ADD_TEST(test_blockwiseconvolution test_convolution.cxx)
+if(NOT THREADING_FOUND)
```

Comment:
It would be easy (if ugly) to add support for single threaded builds to the chunked array classes (in fact, I already had it in an earlier version). Is this extension worthwhile?